packages/create-skill-app/templates/next/package.json

```diff
@@ -37,6 +37,7 @@
     "next-compose-plugins": "^2.2.1",
     "next-sitemap": "^3.0.5",
     "nodemailer": "^6.7.2",
+    "mjml": "^4.12.0",
```

Comment:
this causes a tailspin if not included which is important to note. would like to understand WHY these missing libs affect it that way, but if it's not there it breaks everything

Reply:
weird